### PR TITLE
Reformatting

### PR DESCRIPTION
**Pull request by Felli**
- [x] ~~Review & Insight~~
- [x] Reformatting `/messages/*` / Fixing unneeded lines

### CODE REVIEW

**Comment by Felli** [comment]
@cebe seeings as you have more experience with Yii than I,  could you possibly answer if the closing tag is even necessary within `config.php`?

**Comment by Felli** [comment]
Never mind removing the closing tags will break `/user/account/edit`.

File: config.php [hunk 1]
```diff
@@ -33,3 +33,4 @@
         ]
     ]
 ];
+?>
```
Comment (cebe) [inline config.php]: this is not necessary but does not hurt if it is not followed by a newline like in this case. But if you accidentally add a newline after the `?>` tag PHP will generate output which is sent to the browser which might cause problems with sending headers. So I'd generally recommend not to add closing tags to files that contain only php code.

Comment (Felli) [reply config.php]: Sadly I found that removing the `?>` closing tag causes issues within `/user/account/edit` for some weird reason which is why I re-added the closing tag which seems to have fixed the issue.

**Pull request merged**